packages/frontity-org-theme/src/processors/borders.ts

```diff
@@ -0,0 +1,50 @@
+import { css } from "frontity";
+
+function createBordersProcessors(obj: {
+  [prop: string]: ReturnType<typeof css>;
+}) {
+  return Object.entries(obj).map(([className, style]) => ({
+    name: "borders",
+    test: element => {
```

Comment:
Hey, we changed the API! Now processors receive `({ node, state, libraries, root })` as arguments, so you will have to change `element => ` to `({ node }) => ` and use `node` instead of `element`.

By the way, you can also return implicitly the result of the condition without using `return` to make this simpler, just

```ts
test: ({ node }) =>
  node.type === "element" &&
  node.props.className?.split(/ +/).includes(className)
```